fission/src/ui/modals/configuring/ChangeInputsModal.tsx

```diff
@@ -343,173 +353,139 @@ const ChangeInputsModal: React.FC<ModalPropsImpl> = ({ modalId }) => {
             icon={<FaGamepad />}
             modalId={modalId}
             onAccept={() => {
-                PreferencesSystem.savePreferences()
+                InputSchemeManager.saveSchemes()
                 InputSystem.selectedScheme = undefined
             }}
```

Comment:
Could you delete the robot onCancel() so if the user cancels their input scheme then the robot doesn't spawn

Reply:
Would we maybe want them to be able to spawn a robot and not assign it a control scheme temporarily?

Reply:
@HunterBarclay @Dhruv-0-Arora Is this something we could add to brain config down the line?

Reply:
Yeah